include/contractor/contractor.hpp

```diff
@@ -89,7 +89,8 @@ class Contractor
                           const std::string &geometry_filename,
                           const std::string &datasource_names_filename,
                           const std::string &datasource_indexes_filename,
-                          const std::string &rtree_leaf_filename);
+                          const std::string &rtree_leaf_filename,
+                          const double &verify_weights);
```

Comment:
`verify_weights` does not describe anything resembling a `double` to me. Is this a factor? Do I have to do a verification in every contractor? Can you choose a name that describes the purpose of this? Also, can we add a default parameter value to disable checks if not wanted?

In Addition: passing a double as const reference doesn't improve stack-usage. Passing a const value should be fine.